tools/release-post-generator/lib/contributors.ts

```diff
@@ -0,0 +1,95 @@
+/**
+ * External dependencies
+ */
+import { checkoutRef, sparseCheckoutRepo } from 'code-analyzer/src/git';
+import { readFile } from 'fs/promises';
+import { join } from 'path';
+import semver from 'semver';
+
+/**
+ * Internal dependencies
+ */
+import { ContributorData, getContributorData } from './github-api';
+
+const OTHER_WATCHED_PACKAGES = [
+	{
+		displayName: 'WooCommerce Blocks',
+		packagist: 'woocommerce/woocommerce-blocks',
+		org: 'woocommerce',
+		repo: 'woocommerce-blocks',
+		versionPrefix: 'v',
+	},
+	{
+		displayName: 'ActionScheduler',
+		packagist: 'woocommerce/action-scheduler',
+		org: 'woocommerce',
+		repo: 'action-scheduler',
+		versionPrefix: '',
+	},
+];
+
+export const generateContributors = async (
+	currentVersion: string,
+	previousVersion: string
+) => {
+	const repoPath = await sparseCheckoutRepo(
```

Comment:
Ported the checking of composer files from original, but we use a sparse checkout instead.